plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/internal/MavenDependencyVersionsFetcherFILE.kt

```diff
@@ -0,0 +1,22 @@
+package de.fayard.refreshVersions.core.internal
+
+import de.fayard.refreshVersions.core.ModuleId
+import java.io.File
+
+internal class MavenDependencyVersionsFetcherFILE(
+    moduleId: ModuleId,
+    repoUrl: String
+) : MavenDependencyVersionsFetcher(
+    moduleId = moduleId,
+    repoUrl = repoUrl
+) {
+    private val baseFolder = File(repoUrl)
+
+    init {
+//        require(repoUrl.endsWith('/'))
```

Comment:
There's no reason to leave commented out code. It's confusing to have in a codebase.